examples/private/version.tf

```diff
@@ -0,0 +1,9 @@
+terraform {
+  required_version = ">= v1.3.0, <1.7.0"
```

Comment:
We are no longer restricting upper limit for terraform version, so just change this to `">= 1.3.0"`